src/common/components/MktSideRecommendations.jsx

```diff
@@ -148,30 +188,28 @@ function MktSideRecommendedCourses({ title, endpoint, technologies, containerPad
             {title || t('continue-learning-course')}
           </Heading>
         )}
-        {!isLoading && courses?.length > 0 ? (
+        {!isLoading && recommendations?.length > 0 ? (
           <Box display="flex" flexDirection={{ base: 'row', md: 'column' }} overflow="auto" gridGap="14px">
-            {courses.map((course) => {
-              const courseLink = course?.course_translation?.landing_url;
-              const link = courseLink || `${ORIGIN_HOST}${langConnector}/${course?.slug}`;
-              // const tags = course?.technologies?.length > 0 && typeof course?.technologies === 'string'
-              //   ? course?.technologies?.split(',').map((tag) => toCapitalize(tag?.trim()))
-              //   : [];
+            {recommendations.map((recom) => {
+              const recomLink = getLink(recom);
+              const link = recom?.course_translation?.landing_url ? `${recomLink}?internal_cta_placement=mktsiderecommendedcourses&internal_cta_content=${recom?.slug}&internal_cta_campaign=null`
+                : recomLink;
               const tags = [];
-              // const tags = ['Free course'];
 
               return (
-                <Container border="1px solid" borderColor={{ base: 'default', md: 'success' }} key={course?.slug} course={course} courses={courses} borderRadius={rest.borderRadius} padding={containerPadding}>
+                <Container border="1px solid" borderColor={{ base: 'default', md: 'success' }} key={recom?.slug} course={recom} courses={recommendations} borderRadius={rest.borderRadius} padding={containerPadding}>
                   <TagCapsule tags={tags} background="green.light" color="green.500" fontWeight={700} fontSize="13px" marginY="0" paddingX="0" variant="rounded" gap="10px" display={{ base: 'none', md: 'inherit' }} />
                   <Box display="flex" flexDirection={{ base: 'column', md: 'row' }} gridGap="8px">
                     <TagCapsule tags={tags} background="green.light" color="green.500" fontWeight={700} fontSize="13px" marginY="0" paddingX="0" variant="rounded" gap="10px" display={{ base: 'inherit', md: 'none' }} />
 
-                    <Image display={{ base: 'none', md: 'inherit' }} src={course?.icon_url} width="46px" height="46px" borderRadius="8px" background={course?.color || 'green.400'} />
-                    <Heading as="span" size="18px">
-                      {course?.course_translation?.title}
+                    {recom?.icon_url
```

Comment:
We must include the icon_url of the primary asset technology of the recommended exercise in case it is an exercise

Reply:
`GET /v1/registry/asset?asset_type=EXERCISE&status=PUBLISHED&technologies={comma_separated_technologies}`
returns an array of exercises (objects). however, technologies key is an array of strings.

### example: 
technologies: ['html5']